Makefile

```diff
@@ -10,9 +10,12 @@ USER_AGENT=${GIT_REPO}/${PACKAGE_VERSION}
 
 OPENAPI_IMAGE_TAG=v7.4.0
 OPENAPI_IMAGE=openapitools/openapi-generator-cli:${OPENAPI_IMAGE_TAG}
+GOSWAGGER_IMAGE_TAG=v0.31.0
+GOSWAGGER_IMAGE=ghcr.io/go-swagger/go-swagger:${GOSWAGGER_IMAGE_TAG}
```

Comment:
go-swagger was the tool I ultimately settled on after trying a number of others. The common failing in other "combine" tools was that they left `$ref: spec_1.yaml#/...` in the combined spec documents. The go-swagger `flatten` feature (or attribute of combining) is what was missing in the other tools.